Yearn-agents/InstantWithdrawl/src/agent.ts

```diff
@@ -0,0 +1,57 @@
+import { BlockEvent, Finding, HandleBlock, getJsonRpcUrl } from "forta-agent";
+import axios from "axios";
+import { generateFinding, getAccounts, Mapping } from "./utils";
+import Web3 from "web3";
+import runServer from "./blockchain";
+import { withdrawAbi } from "./abi";
+
+const providerHandleBlock = (Web3: any, axios: any): HandleBlock => {
```

Comment:
I think you can get rid of 'any' types. E.g. (web3:Web3, _axios: axios)

Reply:
There is no Axios type export from Axios. It's only for AxiosResponse and other similar.  Also for web3, that also wont work because, the Web3 variable I am passing in as the argument, is not an Object of Type Web3, but the class Web3 itself.
This is what vscode throws:
`Argument of type 'typeof Web3' is not assignable to parameter of type 'Web3'.
  Type 'typeof Web3' is missing the following properties from type 'Web3': defaultAccount, defaultBlock, currentProvider, setProvider, and 5 more.ts(2345)
agent.ts(55, 36): Did you mean to use 'new' with this expression?`